### ISSUE TITLE
Pandas DataFrame example raises error "No encoder found "

### ISSUE THREAD

**Issue by hamelsmu** [nteract/scrapbook#72]
Running this example from [the docs](https://nteract-scrapbook.readthedocs.io/en/latest/usage-glue.html):

```py
import scrapbook as sb
import pandas as pd

df = pd.DataFrame({"a": [1,2], "b": [3,4]})

sb.glue("hello", "world")
sb.glue("number", 123)
sb.glue("some_list", [1, 3, 5])
sb.glue("non_json", df, encoder='pandas')
sb.glue("some_dict", {"a": 1, "b": 2})
```

Causes this error:

```bash
---------------------------------------------------------------------------
ScrapbookMissingEncoder                   Traceback (most recent call last)
<ipython-input-2-730353fe4323> in <module>()
      7 sb.glue("number", 123)
      8 sb.glue("some_list", [1, 3, 5])
----> 9 sb.glue("non_json", df, encoder='pandas')
     10 sb.glue("some_dict", {"a": 1, "b": 2})

2 frames
/usr/local/lib/python3.6/dist-packages/scrapbook/encoders.py in encode(self, scrap, **kwargs)
    110             raise ScrapbookMissingEncoder(
    111                 'No encoder found for "{data_type}" data type!'.format(
--> 112                     data_type=encoder
    113                 )
    114             )

ScrapbookMissingEncoder: No encoder found for "None" data type!
```

Here is a [reproducible Colab Notebook](https://colab.research.google.com/drive/1k-HCYvmlg1asqH2aqwtKdpa1PR8NZNgc) demonstrating this issue.

**Comment by MSeal** [comment]
I need to release the code that's in latest/master which has this improvement. I'll prioritize getting a release by this weekend out.

**Comment by MSeal** [comment]
@hamelsmu The change is released now, you can see it in version 0.4.1 (your code above works with it).
